packages/offix-datastore/src/storage/adapters/IndexedDBStorageAdapter.ts

```diff
@@ -23,7 +23,7 @@ export class IndexedDBStorageAdapter implements StorageAdapter {
         });
         this.transaction = transaction;
     }
-
+    // TODO Wrong architecture. Store can be created on demand
```

Comment:
This is not exactly accurate but points out API problem. 
I hope to create issue with solution to not resolve too many problems in single PR.